cumulusci/tasks/apex/testrunner.py

```diff
@@ -374,9 +376,8 @@ def _write_output(self, test_results):
                     s += ">\n"
                     s += '    <failure type="failed" '
                     if result["Message"]:
-                        s += 'message="{}">'.format(cgi.escape(result["Message"]))
-                    else:
-                        s += ">"
```

Comment:
@cdcarter this was my other lazy coverage hack.

Once we get line coverage to 100% we can turn on branch coverage and it'll keep us honest.

Reply:
i mean, this is also more aesthetically pleasing

Reply:
First time I've heard that phrase used to describe code that generates xml :)